juriscraper/pacer/reports.py

```diff
@@ -176,6 +176,7 @@ def download_pdf(
         pacer_case_id: str,
         pacer_doc_id: int,
         pacer_magic_num: Optional[str] = None,
+        email_notice_type: Optional[str] = None,
```

Comment:
I think we could simplify this parameter by calling it `appellate` and making it a boolean that's defaulted to False:

```suggestion
        appellate: bool = False,
```

Because I think this applies beyond NDAs vs. NEFs, and as I saw over in CourtListener, it's weird to be saying *anything* about emails when calling this function elsewhere.

Reply:
Yeah, I agree, I've changed and set it to False as default.